README.md

```diff
@@ -14,7 +14,7 @@ The protocol allows a "client" and "server" to establish a bidirectional connect
 
 ## Install
 
-Requires Node >= v8.10.
+Requires Node >= v8.10. Alternatively, `ilp-protocol-stream` will run in the browser via webpack, but only as client.
```

Comment:
I'll update it to say >v10.

Reply:
Seems good to drop node 8 support. @emschwartz do you have any objections to dropping v8 support? I recall when we first released this it was very important to us but I think that was just because at that time it was sill heavily used and we wanted to support i .